guides/common/modules/proc_manually-tuning-db-pool.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Manually_tuning_db_pool_{context}"]
+= Manually tuning db_pool
+
+If you decide not to rely on automatically configured value, you can apply custom number like this:
+
+[options="nowrap", subs="+attributes"]
+----
+# {foreman-installer} --foreman-db-pool 10
+----
+
+Please follow xref:Applying_configurations_{context}[] section to apply the change.
```

Comment:
```suggestion
```

I think this is unnecessary because foreman-install is already run.

Reply:
I wanted the link to the section here, because the section contains a note about backups and health check. I'll make it:
```
. Apply your changes to {ProjectServer}.
For more information, see xref:Applying_configurations_{context}[].
```

Reply:
Great!